stellargraph/datasets/datasets.py

```diff
@@ -707,3 +707,42 @@ def load(self):
             ``graph``.
         """
         return _load_tsv_knowledge_graph(self)
+
+
+class IAEnronEmployees(
+    DatasetLoader,
+    name="IAEnronEmployees",
+    directory_name="ia-enron-employees",
+    url="http://nrvis.com/download/data/dynamic/ia-enron-employees.zip",
+    url_archive_format="zip",
+    expected_files=["ia-enron-employees.edges", "readme.html"],
+    description="A dataset of edges that represent emails sent from one employee to another."
+    "There are 50572 edges, and each of them contains timestamp information. "
+    "Edges refer to 151 unique node IDs in total.",
```

Comment:
> Maybe we could add some mechanism in DatasetLoader to hold onto the citation information? Otherwise any notebook that uses this dataset would have to paste something like that

Instead of forcing the notebooks to paste in the citation manually, we can include it here, like some of the other datasets too, e.g. add an extra line

```python
"Ryan A. Rossi and Nesreen K. Ahmed “The Network Data Repository with Interactive Graph Analytics and Visualization” (2015)"
```

If the notebook then does `display(HTML(dataset.description))` like most do, it'll include this info, as will our documentation.

Maybe it'd be good to expand this to have an actual "citation" field too.